lib/static.ts

```diff
@@ -53,11 +57,21 @@ export async function analyzeStatically(
   const excludeBaseImageVulns = isTrue(options["exclude-base-image-vulns"]);
 
   const names = getImageNames(options, imageName);
+  let ociDistributionMetadata: OCIDistributionMetadata | undefined;
+  if (options.imageNameAndTag && options.digests?.manifest) {
+    ociDistributionMetadata = constructOCIDisributionMetadata({
+      imageName: options.imageNameAndTag,
+      manifestDigest: options.digests.manifest,
+      indexDigest: options.digests.index,
+      platform: options.platform ?? "linux/amd64",
```

Comment:
is this because the default platform pulled is `linux/amd64`?

Reply:
Yes, we need to be careful when we consume this library though to set platform appropriately - we don't want to pull something exotic and then have this default to `"linux/amd64"`. I can't really avoid a default here though without a breaking change to the API which doesn't feel necessary.

Reply:
iirc the default is set to the arch of the machine the scan runs on, which is why Apple chip users need to set `--platform=linux/amd64` when running a test/monitor. This is done in the CLI and passed as an option to `scan()` I think. Be careful if this overrides that behaviour as I would class that as a breaking change.